### DISCUSSION TITLE
Broadcasting to chat?

### DISCUSSION THREAD

**Discussion by sjhu2020** [GriefPrevention/GriefPrevention#1957] [Q&A]
I've been using GriefPrevention for a while now, recently we've become concerned because it seems to be announcing possible exploitive players and randomly inspecting players for hacked clients, are these even a feature?  Because we've never been able to reproduce the announcement on our other servers, so we're worried someone is doing something malicious.  If not, is there a way to disable this? 
![image](https://user-images.githubusercontent.com/74697916/196601155-d68ad76a-7531-47e1-8f03-ea5a4f61c84e.png)
<img width="601" alt="image" src="https://user-images.githubusercontent.com/74697916/196601215-d30fedcd-aa35-417f-a865-520308793e34.png">
This player was an OP and was using a completely vanilla client
![image](https://user-images.githubusercontent.com/74697916/196601274-1b1f1863-d384-4dc3-9ce4-65c326b7edd1.png)
And this was me, also OP'ed

**Comment by Jikoo** [comment]
This is not a feature of GP. GP is not an anti-cheat and contains no anti-cheat features. Those look like Essentials' broadcasts, too, so whatever is causing it is taking a very quick and lazy route. Would recommend you check all of your plugins, not just GP.

**Reply by sjhu2020** [reply]
Thank you, we do suspect that it is another plugin pretending to be GP.  We included a few extra plugins on this server and made sure to remove them, we will be sure to report the plugins once we have confirmed that this worked.
